README.md

```diff
@@ -65,6 +65,7 @@ These are all the permissions that you can specify:
 ### Data Page
 - `data_uuids`: User can view the UUIDs data in the Data page.
 - `data_trips`: User can view the trips data in the Data page.
+- `data_demographics`: User can view the demographics data in the Data page.
```

Comment:
you should add an entry for `data_demographics_columns_exclude` now that you have added that config option.